src/wallet/rpc/backup.cpp

```diff
@@ -225,7 +227,8 @@ RPCHelpMan importaddress()
             "\nNote: If you import a non-standard raw script in hex form, outputs sending to it will be treated\n"
             "as change, and not show up in many RPCs.\n"
             "Note: Use \"getwalletinfo\" to query the scanning progress.\n"
-            "Note: This command is only compatible with legacy wallets. Use \"importdescriptors\" with \"addr(X)\" for descriptor wallets.\n",
+            "Note: This command will create a new descriptor/s if the wallet has private keys disabled and has no legacy scriptPubKey manager."
```

Comment:
Suggested alternative wording that avoids "scriptPubKey manager".

```
For descriptor wallets, this command only works if it has private keys disabled.
It will create a new descriptor/s. Otherwise, use \"importdescriptors\" instead.
```

Reply:
pushed without the last part, mainly because the user cannot import watch-only descriptors on a private keys enabled wallet. So, `importdescriptors` wouldn't help anyway.